tooling/nargo_fmt/src/formatter/expression.rs

```diff
@@ -230,12 +232,25 @@ impl<'a, 'b> ChunkFormatter<'a, 'b> {
 
         group.text(params_and_return_type_chunk);
 
-        let body_is_block = matches!(lambda.body.kind, ExpressionKind::Block(..));
+        let (body_is_block, block_statements_count) =
+            if let ExpressionKind::Block(block) = &lambda.body.kind {
+                (true, block.statements.len())
+            } else {
+                (false, 0)
```

Comment:
Since body_is_block always implies block_statements_count, perhaps we should use an option instead? `block_statement_count: Option<usize>`?

Reply:
I considered this but wasn't sure which option to use (no pun intended :-D). But given that you are suggesting it too, I'll do the change.